dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListenerManager.cpp

```diff
@@ -9,26 +9,24 @@ void NotificationListenerManager::Initialize(std::shared_ptr<ForegroundSinkManag
     m_toastRegistrationManager = toastRegistrationManager;
 }
 
-void NotificationListenerManager::SetAppIdMapping(std::map<std::wstring, std::wstring>& appIdList)
+void NotificationListenerManager::SetAppIdMapping(std::map<std::wstring, std::pair<std::wstring, winrt::guid>>& appIdList)
```

Comment:
This argument desperately needs a comment so that devs understand what exactly is being mapped.

Something along the lines of:
ProcessName->(appId, comclsid)

Please add comments when code isn't self explanatory